pkg/parser/model/ddl.go

```diff
@@ -1199,6 +1199,9 @@ type SchemaDiff struct {
 	OldSchemaID int64 `json:"old_schema_id"`
 	// RegenerateSchemaMap means whether to rebuild the schema map when applying to the schema diff.
 	RegenerateSchemaMap bool `json:"regenerate_schema_map"`
+	// RecoverSnapshotTS is set when the diff is too large to be saved in SchemaDiff.
+	// infoschema should use this TS to read meta directly.
+	RecoverSnapshotTS uint64 `json:"recover_snapshot_ts,omitempty"`
```

Comment:
what if: after flashback schema job to done, and before reload, GC remove all those keys

Reply:
I remember it will fallback to full ode

https://github.com/pingcap/tidb/blob/e858f55f62321b94be2f4962c3c4524a4505d2a6/pkg/domain/domain.go#L345

Reply:
we can reload this schema only if we use current ts